Tools/MobileMiniBrowser/MobileMiniBrowser.xcodeproj/project.pbxproj

```diff
@@ -441,11 +449,18 @@
 			isa = PBXTargetDependency;
 			platformFilters = (
 				ios,
-				xros,
```

Comment:
I don't think that this was mentioned in the commit message, what does this do?

Reply:
Good catch, I think it's a mistake. I used an Xcode bundle which did not have a visionOS SDK installed.